src/server/misc/cmds/cmds_1.20.go

```diff
@@ -0,0 +1,22 @@
+//go:build go1.20
+
+package cmds
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"syscall"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/log"
+	"go.uber.org/zap"
+)
+
+func setupControl(d *net.Dialer) {
+	d.ControlContext = func(ctx context.Context, network, address string, c syscall.RawConn) error {
+		var fd uintptr
+		c.Control(func(f uintptr) { fd = f }) //nolint:errcheck
```

Comment:
why's this implementation 1.20 specific?

Reply:
Hahah brendon asked the same thing on the original PR.  ControlContext is newly-added in 1.20.  There is just Control in 1.19, which doesn't interact with our logger as nicely.  In retrospect, it's not worth 2 files just to print the FD number (but the FD number is nice when stracing, which maybe someone will do sometime, which is why I added it) 😂